facades/translation.go

```diff
@@ -0,0 +1,11 @@
+package facades
+
+import "github.com/goravel/framework/contracts/translation"
+
+func Translation() translation.Translation {
```

Comment:
S: It's `Lang` in Laravel: https://github.com/laravel/framework/blob/c262c1ffce93c5fa4b9e00da1a54d8c02ddddcf6/src/Illuminate/Support/Facades/Lang.php#L44, and it seems that this method needs to add a `context.Context` parameter.